providers/cloudflare/cloudflareProvider.go

```diff
@@ -72,15 +72,16 @@ type cloudflareProvider struct {
 	cfClient        *cloudflare.API
 }
 
-func labelMatches(label string, matches []string) bool {
-	printer.Debugf("DEBUG: labelMatches(%#v, %#v)\n", label, matches)
-	for _, tst := range matches {
-		if label == tst {
-			return true
-		}
-	}
-	return false
-}
+// TODO: remove this function after deleting all commented code referecing it
```

Comment:
Change `TODO` to `TODO(dlemenkov)`  (Always sign your TODOs.  It doesn't mean you have to fix the TODO, but it makes it easier to know who inserted it.)